Client/Frontend/Home/ActivityStreamPanel.swift

```diff
@@ -44,15 +42,24 @@ class ActivityStreamPanel: UITableViewController, HomePanel {
         return UILongPressGestureRecognizer(target: self, action: #selector(ActivityStreamPanel.longPress(_:)))
     }()
 
+    // Not used for displaying. Only used for calculating layout.
+    lazy var topSiteCell: ASHorizontalScrollCell = {
+        let customCell = ASHorizontalScrollCell(frame: CGRect(x: 0, y: 0, width: self.view.frame.size.width, height: 0))
+        customCell.delegate = self.topSitesManager
+        return customCell
+    }()
+
```

Comment:
This is probably my least favorite change. I create an extra TopSiteCell in order to do height calculations.